### ISSUE TITLE
Glyphs Source Format concerns ahead of making Gurage additions

### ISSUE THREAD

**Issue by NeilSureshPatel** [googlefonts/googlesans#532]
I am attaching some test files to see if there are any issues with using Glyphs 2 formatted sources saved out of Glyphs 3. The original GS Ethiopic work was done using Glyphs 2.

There are 3 files.
Original File created and saved in Glyphs 2:          GoogleSansEthi-Draft-G2.glyphs
File saved out of Glyphs 3 into Glyphs 2 format:   GoogleSansEthi-Draft-G3toG2.glyphs
File saved out of Glyphs 3 into Glyphs 3 format:   GoogleSansEthi-Draft-G3toG3.glyphs

[Glyphs Format Testing.zip](https://github.com/googlefonts/googlesans/files/9649152/Glyphs.Format.Testing.zip)

**Comment by chrissimpkins** [comment]
@belluzj I recognize that you are tied up in the v10 release work, but if you happen to have any initial concerns about delivery of a Gurage expansion delivered in Glyphs 3 format for merge here to pass along to @NeilSureshPatel will you please let him know in this thread?  We just want to make sure that Neil and Mark don't get too far down the road in Glyphs 3 editor development if we will be unable to support the import during Q4.

Ty!

**Comment by chrissimpkins** [comment]
#542

**Comment by belluzj** [comment]
@chrissimpkins Indeed we haven't started yet on account of the other v10 work but we'll have a look as soon as possible. It's on our TODO list.

**Comment by madig** [comment]
The G2 source fails to import because the default source/master does not sit at the default location.

**Comment by madig** [comment]
@NeilSureshPatel please make sure the master and instance locations match Google Sans, plus the weight mapping. Only then can I do a proper import test.

**Comment by NeilSureshPatel** [comment]
@madig I'll fix the default instances and masters. I wanted to clarify what you mean by "locations". Are you just talking about the axis set point and names or do you want me to open a branch and place the files in the correct source directory tree also? Thanks.

**Comment by madig** [comment]
Oh, don't worry about file location. I mean location of masters and instances on the axes, plus axis mapping (`avar`).

**Comment by NeilSureshPatel** [comment]
Just realized these sources don't have variable font instances so there are no `avar` settings. I do have the axes coordinates set correctly though.

**Comment by madig** [comment]
Please add some instances then, just for testing, so that the axis mapping lines up.

**Comment by NeilSureshPatel** [comment]
Ok. I am not sure if this is all correct, but hopefully it works. The Glyphs 2 formatted files don't have Variable Font instances since Glyphs 2 didn't handle variable fonts in the same way. Variable fonts were created from the export menu in Glyphs 2 with no special setup. The Glyphs 3 formatted file does have a Variable Font instance with the axis mapping setup. Let me know if this works.
[Glyphs Format Testing.zip](https://github.com/googlefonts/googlesans/files/9747995/Glyphs.Format.Testing.zip)

**Comment by Hoolean** [comment]
Hello Neil! Thank you for the updated sources; I will take a look now : )

**Comment by Hoolean** [comment]
@NeilSureshPatel

In the Glyphs 2 sources, there are brace layers under Text Regular. Would you be able to tell us if these are necessary for the design and, if so, which axis are the 250 and 500 positions referring to? (in particular, 250 seems to be outside the bounds of both)

![image](https://user-images.githubusercontent.com/2150762/195330696-15abd0ab-a391-4d2d-9962-846389b8f4b9.png)

**Comment by NeilSureshPatel** [comment]
Those brace layers not necessary for the deign. Sorry, I forgot those were in there. They are artifacts from early prototype work on Google Sans Ethiopic. Those can be safely removed.

**Comment by Hoolean** [comment]
Thank you for the speedy response Neil : ) On removing the brace layers, everything worked smoothly.

@MariannaPaszkowska @chrissimpkins @NeilSureshPatel 

I am happy to say that we encountered no issues importing any of the test sources, and so have confidence that we can import sources from or derived from Glyphs 3 (using the same subset of features) in the future, without adjusting our existing process.

While we observed some superficial differences between the staging UFOs produced by glyphsLib¹, none of these propagated through our normal import workflow to the merged font, as they were not in places we import.

The only area we saw inconsistent behaviour was with brace layers, which were broken in all cases. This is not a regression, however, as sparse layers are unimplemented in our merge process for both versions of Glyphs.

---

¹The unimported differences included:
- Whitespace in feature files
- `postscript*` properties in fontinfo.plist
- Style names in designspace
- Diff in non-Ethiopic glyph files:
  - In the sample of these we checked manually, glyphsLib added "com.schriftgestaltung.Glyphs.ComponentInfo" in the v3-derived sources; this property would have been merged in if present in Ethiopic files, but we would not expect this to have any effect on the final TTFs
- Extra instance in g3tog3 designspace
